set.mm

```diff
@@ -389459,34 +389459,27 @@ then any point of the half line ( ` R ` ` A ` ) also lies opposite to
-        PXSXTVLZVMAUUFUUGWOYDYFYAACXKCLFIJKNPOQSUUISUDACLFXIIJYRXJKNPOYSYTQSUUH
-        UDVJVKVEVNYHXKCDXMFXOGXNIJKNPOUUEYLUUJYNYOUUBYQUUDUUKVQYHLDYJXNFIJKNPOY
-        LYMYOUUCUUDYHDCLXNFXOGYJIJKNPOUUEYLYOYNYMUUDYQUUCYHLCDYJFXOGXNIJKNPOUUE
-        YLYMYNYOUUCYQUUDYHXKXJUNZCDUOZYBYJGXNUOXOAUUMYBWOYDYFYAAUULCDDLCACLFXII
-        JYRXJKNPOYSYTQSUUHUDVRZACVSADVSVTVEYHXKCDFXOXIIJYRXJKYIXMGXNNPOYSYTYLUU
-        EUUHUUAUUJYNUUBYQYOUUDAXKCWAYDYFYAACLFXIIJYRXJKNPOYSYTQSUUHUDACLUIWBWCZ
-        VEUUKWDWEWFWGVKWHYHMYJGFIJXQVFNOYKYPUUCYQYLYHGFXIIJXQYRYIKMYJGNPOYSYTYL
-        UUAYKYQYPUUCYQYHGMAGMWAYDYFYAUJVEWBYHGXMFXIIJYRYIKNPOYSYTYLYQUUAUUBYHXM
-        EGFIJXQVFNOYKUUBAEFVBYDYFYAUAVEZYQYLYGXPXSXTWIZWJWCYHGMXMJVPMYJYIUNZJVP
-        YHXMGMFIJKNPOYLUUBYQYPYHEXMMGFIJXQNOYKUUPUUBYPYLYQYHXMEGFIJXQVFNOYKUUBU
-        UPYQYLUUQWKAGEMJVPVBYDYFYAUHVEWLWMYHUURXMMJYHGXMFXIIJYRYIKNPOYSYTYLYQUU
-        AUUBVRWNWPWQWKYGXPXSXTWRWSAXLEGHUOYCURYAULFWTUKFWTAXKCDBFGCDIEJHXQNOQYK
-        UUISTRSTAXKCDXKFCDIJXQBNOYKQUUISTUUISTAXKCDFIJXQNOQYKUUISTUUOABCDEFGHIJ
-        XQNOYKQRSTUAUBUCUFXAXBRACFXIIJXQYRXJKBXKBNPOYSYTQUUHYKSRUUIRABCDEFGHIJX
-        QNOYKQRSTUAUBUCUFXCUUOACBLJVPBUULJVPUGAUULLBJUUNWNWPWQXDUAUBUCUFXEAUKUL
-        XKCDEFGHIJXQNOYKQUUISTUAUBUCXFXGXH $.
+        cstrkg clng simpllr mircl simplr mirmir eqidd s3eqd necomd mirne simpr1
+        wceq mirtrcgr eqbrtrrd simpr2 hlne1 co hlcomd btwnhl tgbtwncom eleqtrrd
+        oveq2d mirhl2 simpr3 iscgrad wrex cgrane2 cgraid cgrane1 cgrahl1 cgratr
+        wne iscgra mpbid r19.29vva ) ALCIUMUNZUNZUNZCDUOZUKUPZGULUPZUOIUQUNZURZ
+        XEEGIUSUNZUNZURZXFHXJURZUTZLCDUOZMGHUOIVAUNZURUKULFFAXEFVBZVCZXFFVBZVCZ
+        XMVCZLCDMFGHIJXIXEGXAUNZUNZXFNOXIVDZAIVFZVBXPXRXMQVEZALFVBXPXRXMUDVEACF
+        VBXPXRXMSVEZADFVBXPXRXMTVEZAMFVBXPXRXMUEVEZAGFVBXPXRXMUBVEZAHFVBXPXRXMU
+        CVEXTGFXAIJIVGUNZYAKXENPOYJVDZXAVDZYEYIYAVDZAXPXRXMVHZVIZXQXRXMVJZXTXCX
+        BUNZCDUOZXNYBGXFUOXGAYRXNVQXPXRXMAYQCDDLCACLFXAIJYJXBKNPOYKYLQSXBVDZUDV
+        KZACVLADVLVMVEXTXCCDFXGXAIJYJXBKYAXEGXFNPOYKYLYEXGVDYSYMAXCFVBXPXRXMACF
+        XAIJYJXBKLNPOYKYLQSYSUDVIZVEYFYNYIYGYPAXCCWQXPXRXMACLFXAIJYJXBKNPOYKYLQ
+        SYSUDACLUIVNVOZVEXSXHXKXLVPVRVSXTMYBGFIJXIYDNOYCYHYOYIYEXTGFXAIJXIYJYAK
+        MYBGNPOYKYLYEYMYCYIYHYOYIXTGMAGMWQXPXRXMUJVEVNXTGXEFXAIJYJYAKNPOYKYLYEY
+        IYMYNXTXEEGFIJXIYDNOYCYNAEFVBXPXRXMUAVEZYIYEXSXHXKXLVTZWAVOXTGMXEJWBMYB
+        YAUNZJWBXTXEGMFIJKNPOYEYNYIYHXTEXEMGFIJXINOYCUUCYNYHYEYIXTXEEGFIJXIYDNO
+        YCYNUUCYIYEUUDWCAGEMJWBVBXPXRXMUHVEWDWEXTUUEXEMJXTGXEFXAIJYJYAKNPOYKYLY
+        EYIYMYNVKWGWFWHWCXSXHXKXLWIWJAXDEGHUOXOURXMULFWKUKFWKAXCCDBFGCDIEJHXINO
+        QYCUUASTRSTAXCCDXCFCDIJXIBNOYCQUUASTUUASTAXCCDFIJXINOQYCUUASTUUBABCDEFG
+        HIJXINOYCQRSTUAUBUCUFWLWMRACFXAIJXIYJXBKBXCBNPOYKYLQYSYCSRUUARABCDEFGHI
+        JXINOYCQRSTUAUBUCUFWNUUBACBLJWBBYQJWBUGAYQLBJYTWGWFWHWOUAUBUCUFWPAUKULX
+        CCDEFGHIJXINOYCQUUASTUAUBUCWRWSWT $.
     $}
```

Comment:
keep the old proof as sacgrOLD.  For details see my comment in the conversations part.